ostelco-ios-client/Models/Region.swift

```diff
@@ -6,6 +6,14 @@
 //  Copyright © 2019 mac. All rights reserved.
 //
 
+func getSimProfileFromSimProfilesArray(_ simProfiles: [SimProfile]) -> SimProfile? {
+    if simProfiles.isNotEmpty {
+        return simProfiles[0]
```

Comment:
i would just do `return simProfiles.first` here - that returns `nil` if the array is empty

Reply:
will do!